### PR TITLE
Frontend Layout

### PR DESCRIPTION
**Pull request by ghost**
I finished frontend layout.
@at-dungnh  please review for me.

### CODE REVIEW

File: resources/views/frontend/dashboard/index.blade.php [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                    <div class="row">
+                        <div class="col-lg-3 col-md-4 col-sm-7">
+                            <div class="post-card">
+                                <a href="#" class="entry-thumb-link">
+                                    <div class="entry-thumb-wrapper">
+                                        <img src="{!! config('define.image_default') !!}">
+                                        <div class="entry-thumb-overlay"></div>
+                                    </div>
+                                </a>
+                                <div class="entry-meta">
```
Comment (dzungnguyenh) [inline resources/views/frontend/dashboard/index.blade.php:18]: Will this block update late?

File: resources/views/frontend/layouts/partials/footer.blade.php [hunk 2]
```diff
@@ -0,0 +1,21 @@
+
+<footer class="footer-copy">
+    <div class="container">
+      <div class="row">
+        <div class="col-sm-4">
```
Comment (dzungnguyenh) [inline resources/views/frontend/layouts/partials/footer.blade.php:5]: [rank C] Please put these test to language file

File: resources/views/frontend/layouts/partials/header.blade.php [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Bootswatch: Sandstone</title>
```
Comment (dzungnguyenh) [inline resources/views/frontend/layouts/partials/header.blade.php]: [rank C] What does title mean?

File: resources/views/frontend/layouts/partials/navigation.blade.php [hunk 4]
```diff
@@ -0,0 +1,44 @@
+<div class="navbar navbar-default navbar-fixed-top">
+    <div class="container">
+        <div class="navbar-header">
+            <a href="../" class="navbar-brand">Promotions</a>
```
Comment (dzungnguyenh) [inline resources/views/frontend/layouts/partials/navigation.blade.php]: [rank C] please check this text

File: resources/views/frontend/layouts/partials/side_bar.blade.php [hunk 5]
```diff
@@ -0,0 +1,9 @@
+<div class="col-lg-3 col-md-4 col-sm-5">
```
Comment (dzungnguyenh) [inline resources/views/frontend/layouts/partials/side_bar.blade.php:1]: [rank C] all texts need to put in language files, please

File: resources/views/frontend/dashboard/index.blade.php (hunk 1 above)
Comment (ghost) [inline resources/views/frontend/dashboard/index.blade.php:18]: Yes, this here will code list promotion featured, new, follow...

**Comment by dzungnguyenh** [comment]
Perfect. Thanks.

**Pull request merged**